db/migrate/20240302174817_create_messages.rb

```diff
@@ -0,0 +1,8 @@
+class CreateMessages < ActiveRecord::Migration[7.1]
+  def change
+    create_table :messages do |t|
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] Create a table for storing your messages as stated in the requirement, the greetings column is missing from the schema.rb file. This made me to get a null value from the interface. 👍